src/components/TextInput/TextInput.css

```diff
@@ -0,0 +1,40 @@
+* {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+    text-decoration: none;
+    font-weight: 400;
+    color: white;
+}
+
+html {
+    background: radial-gradient(rgb(255, 230, 4), 1px, rgb(245, 173, 17));
+    background-attachment: fixed;
+    background-size: cover;
+    background-repeat: no-repeat;
+}
```

Comment:
Essas regras estão muito genéricas, você pode explicar melhor o objetivo da declaração do todos com o "*" e do "html"?
Se for realmente pra ser aplicado a todos e ao html como um todo, temos que acrescentar essa regra no App.css

Reply:
Ok .. O "*" serve para definir uma breve formatação das configurações da página do browser, assim não tenho a adição de medidas que o navegador gera por padrão.

Reply:
O seletor "HTML" é para interagir na camada que está abaixo da "body", assim consigo sobrepor os estilos de forma hierárquica para fazer as alterações sem interferir nas demais configurações.